### PR TITLE
Accumulate exercise added

### PR DESCRIPTION
**Pull request by KalebHawkins**
Added accumulate exercise files. I think I got everything right on this one but not entirely sure.﻿

### CODE REVIEW

**Review by kotp — DISMISSED** [review]

**Review by kotp — COMMENTED** [review]
The `description.md` file in both places are identical.  Something seems askew.

**Comment by KalebHawkins** [comment]
For the `description.md` and `instructions.md` what do we need to do with those? Should the `description.md` be a compliment to the `instructions.md` or vice versa?

**Comment by kotp** [comment]
> For the `description.md` and `instructions.md` what do we need to do
> with those? Should the `description.md` be a compliment to the
> `instructions.md` or vice versa?

Look at the [practice-exercises][1] documentation.  It has details
there.

  [1]: https://github.com/exercism/docs/blob/main/building/tracks/practice-exercises.md

File: exercises/practice/accumulate/description.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Description
```
Comment (ErikSchierboom) [inline exercises/practice/accumulate/description.md]: This file should be removed, as exercises don't have a description.md file (the new file is `.docs/instructions.md`, which you've added).

File: exercises/practice/accumulate/Get-Accumulation.tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,23 @@
+BeforeAll {
+    . "./Get-Accumulation.ps1"
+}
+
+Describe 'Test Get-Accumulation.ps1' {
```
Comment (ErikSchierboom) [inline exercises/practice/accumulate/Get-Accumulation.tests.ps1]: Just to be safe: did you consider implementing all test cases in the exercise's canonical data https://github.com/exercism/problem-specifications/blob/main/exercises/accumulate/canonical-data.json? It's perfectly fine not to implement all if they don't make sense for this track, but I did want to give a heads-up.

Comment (KalebHawkins) [reply exercises/practice/accumulate/Get-Accumulation.tests.ps1]: I did consider implementing all test cases. I think that will be in the next PR. For now I wanted to get a basic exercise built and accepted and merged to get a better grasp of the process of building the exercises and what to look for.

Comment (ErikSchierboom) [reply exercises/practice/accumulate/Get-Accumulation.tests.ps1]: Sure!

**Review by ErikSchierboom — APPROVED** [review]

**Review by ErikSchierboom — CHANGES_REQUESTED** [review]
There is one thing still missing: an example implementation that passes all the tests. It should be located at `.meta/example.ps1` (according to the `config.json`).

**Comment by KalebHawkins** [comment]
@ErikSchierboom there is an `example.ps1` in the `.meta/` directory. It passes the tests. 

The config.json shows this:

```json
"example": [
      "%{pascal_slug}.example.ps1"
    ],
    "exemplar": [
      ".meta/%{pascal_slug}.exemplar.ps1"
    ]
```

Should I rename `example.ps1` =>  `examplar.ps1`?
Or should this line be `"%{pascal_slug}.example.ps1"` => `".meta/%{pascal_slug}.example.ps1"` in the `config.json`?

**Comment by kotp** [comment]
The `exemplar` entry is for concept exercises, the `example` file is for practice exercises.  This exercise is practice, so `example`.

But it surely should be `powershell` instead of `pascal`.

**Comment by KalebHawkins** [comment]
@ErikSchierboom can I get reviewed once more. Changes were made.

**Review by ErikSchierboom — APPROVED** [review]
LGTM!

**Pull request merged**